helm/templates/config-service-config.yaml

```diff
@@ -22,6 +22,7 @@ data:
         }
       }
       publish.change.events = {{ .Values.configServiceConfig.publishChangeEvents }}
+      max.label.application.rules.per.tenant = {{ .Values.configServiceConfig.maxLabelApplicationRulesPerTenant }}
```

Comment:
You will need to add maxLabelApplicationRulesPerTenant in values.yaml as well

Reply:
https://github.com/Traceableai/helm-values/pull/1005

Reply:
I meant in values.yaml in this repo `helm/values.yaml`